packages/langium/src/lsp/document-symbol-provider.ts

```diff
@@ -68,8 +71,10 @@ export class DefaultDocumentSymbolProvider implements DocumentSymbolProvider {
         return undefined;
     }
 
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    protected getSymbolKind(type: string): SymbolKind {
+    /**
+     * @deprecated Use the `NodeKindProvider` service instead.
```

Comment:
I think we can remove this unless you want to add the change to v1.3.0. Note that users won't easily notice the deprecation because the TypeScript editor doesn't indicate that you're overriding a deprecated method.